Essentials/src/main/java/net/essentialsx/api/v2/events/TeleportRequestAcceptEvent.java

```diff
@@ -0,0 +1,67 @@
+package net.essentialsx.api.v2.events;
+
+import com.earth2me.essentials.IUser;
+import org.bukkit.event.Cancellable;
+import org.bukkit.event.Event;
+import org.bukkit.event.HandlerList;
+
+/**
+ * Called when a player accepts a teleport.
+ */
+public class TeleportRequestAcceptEvent extends Event implements Cancellable {
+    private static final HandlerList handlers = new HandlerList();
+
+    private final net.ess3.api.IUser requestee;
+    private final net.ess3.api.IUser requester;
+    private final IUser.TpaRequest tpaRequest;
+    private boolean canceled = false;
+
+    public TeleportRequestAcceptEvent(net.ess3.api.IUser requestee, net.ess3.api.IUser requester, IUser.TpaRequest tpaRequest) {
```

Comment:
Maybe change this to `TeleportRequestResponseEvent` and fire on both accept and deny, so plugins can listen to both types of response?